Jenkinsfile

```diff
@@ -13,5 +26,20 @@ pipeline {
                 archiveArtifacts artifacts: '**/build/reports/**', fingerprint: true
             }
         }
+        stage('Build Application') {
+            steps {
+                sh './gradlew build'
+            }
+        }
+        stage('Build and verify images') {
+            steps {
+                sh './packer/verify'
```

Comment:
`packer validate` doesn't run serverspec. It only checks to see if the syntax and configuration is correct.